tests/test_data_models/test_notebook.py

```diff
@@ -243,7 +248,7 @@ def test_notebook_sub(self) -> None:
                 cell_type="markdown",
             ),
         ]
-        assert diff.resolve(keep_first_cells=None) == notebook
+        assert diff.resolve(keep_first_cells=None).dict() == notebook.dict()
```

Comment:
We cannot directly compare notebooks anymore?
Can we add a `__eq__` if that's the case?

Reply:
I'll fix this (exactly with __eq__), it's a leftover from the debugging haha